src/components/organisms/news/news.html

```diff
@@ -0,0 +1,42 @@
+<section>
```

Comment:
```suggestion
<section class="news">
```
各コンポーネントの一番外側の要素には、特定可能な class や id をつけておくとCSSのセレクタが書きやすくなる気がします。